File: Nov2020/16Nov/OddOneOut.cpp
/*
Problem Name: Odd one Out
Problem Link: https://www.hackerearth.com/practice/data-structures/arrays/1-d/practice-problems/algorithm/find-the-odd/
*/

#include<bits/stdc++.h>
using namespace std;
int main(){
    ios::sync_with_stdio(0);    
    cin.tie(0);             
    int n;
    cin>>n;
    int cards[n-1];
    for(int i=0;i<n-1;++i){
        cin>>cards[i];
    }
    sort(cards,cards+n-1);
    for(int j=0,i=1;i<2*n;i=i+2,j++){
        if(cards[j]!=i){
            cout<<i;
            break;
        }
    }
    return 0;
}

File: Nov2020/16Nov/OddOneOut.java
/*
Problem Name: Odd one Out
Problem Link: https://www.hackerearth.com/practice/data-structures/arrays/1-d/practice-problems/algorithm/find-the-odd/
*/
import java.util.*;
import java.io.*;
class OddOneOut {
    public static void main(String args[] ) throws Exception {
        FastReader in = new FastReader(); 
        long n = in.nextLong(), sum = 0;
        if(n == 1)
            System.out.println(1);
        else{
            for(int i=0;i<n-1;i++)
                sum += in.nextLong();
            System.out.println(n*n - sum);
        }
    }
    //for fast io
    static class FastReader 
    { 
        BufferedReader br; 
        StringTokenizer st; 
  
        public FastReader() 
        { 
            br = new BufferedReader(new
                     InputStreamReader(System.in)); 
        } 
  
        String next() 
        { 
            while (st == null || !st.hasMoreElements()) 
            { 
                try
                { 
                    st = new StringTokenizer(br.readLine()); 
                } 
                catch (IOException  e) 
                { 
                    e.printStackTrace(); 
                } 
            } 
            return st.nextToken(); 
        } 
  
        int nextInt() 
        { 
            return Integer.parseInt(next()); 
        } 
  
        long nextLong() 
        { 
            return Long.parseLong(next()); 
        } 
  
        double nextDouble() 
        { 
            return Double.parseDouble(next()); 
        } 
  
        String nextLine() 
        { 
            String str = ""; 
            try
            { 
                str = br.readLine(); 
            } 
            catch (IOException e) 
            { 
                e.printStackTrace(); 
            } 
            return str; 
        } 
    } 
}
